### PR TITLE
Unpark validator automatically

### LINKED ISSUE #344: Unpark validators automatically
Validators should be able to recognize if they have been parked in the staking contract so that they can send an unpark transaction and prevent being automatically inactivated.

### PR DESCRIPTION
**Pull request by vlvrd**

### CODE REVIEW

File: validator/src/validator.rs [hunk 1]
```diff
@@ -127,9 +132,13 @@ impl<TNetwork: Network, TValidatorNetwork: ValidatorNetwork>
         let network1 = Arc::clone(&network);
         let (proposal_sender, proposal_receiver) = ProposalBuffer::new();
 
+        // FIXME: Placeholder, update this line :point_down: as soon as the staking contract changes are merged
+        let id = ValidatorId::default();
```
Comment (sisou) [inline validator/src/validator.rs]: The staking contract changes are merged, so this should be possible to do correctly now?

Comment (vlvrd) [reply validator/src/validator.rs]: It is, this branch is outdated (it was updated before the staking contract changes went into _albatross_)

File: client.toml [hunk 2]
```diff
@@ -14,6 +14,7 @@ port = 8648
 level = "debug"
 
 [validator]
-#wallet_account = ""
-#wallet_password = ""
+validator_address = "4204 2042 0420 4204 2042 0420 4204 2042 0420"
```
Comment (brunoffranca) [inline client.toml]: This doesn't seem like a valid address.

File: validator/src/validator.rs [hunk 3]
```diff
@@ -62,12 +67,15 @@ struct ProduceMicroBlockState {
...
-    // TODO: Also have the validator ID here.
+
     signing_key: bls::KeyPair,
-    wallet_key: Option<keys::KeyPair>,
     database: Database,
     env: Environment,
 
+    validator_address: Address,
+    cold_key: KeyPair,
+    warm_key: KeyPair,
```
Comment (brunoffranca) [inline validator/src/validator.rs:77]: I'm going to need methods to fetch these. Just like the signing_key() method.

**Pull request merged**